heroic-elasticsearch-utils/src/main/java/com/spotify/heroic/elasticsearch/DistributedRateLimitedCache.java

```diff
@@ -94,7 +94,7 @@ public boolean acquire(K key, final Runnable cacheHit) {
       } catch (TimeoutException e) {
         span.setStatus(Status.INTERNAL.withDescription(e.getMessage()));
         memcachedReporter.reportMemcachedTimeout();
-        log.error("Failed to get key from memecached");
+        log.debug("Failed to get key from memecached");
```

Comment:
How come you made this a debug statement? It looks like a real error trying to communicate with memcached.

Reply:
to reduce the spam in the logs - found this less important now that we have a metric reporting the same thing. 

iirc this is mostly happening because we don't use the async framework properly in this module, as this error will be raised when it hits `.toCompletableFuture().get(100, TimeUnit.MILLISECONDS)`